### ISSUE TITLE
Error on clean Windows 10 installation

### ISSUE THREAD

**Issue by GeorgeK1ng** [lee-soft/ViStart#36]
Probably cause in PopulateItemsFromCollection()

![image](https://user-images.githubusercontent.com/98261225/227930889-cf583173-228c-46cf-ad78-72a6d735e04f.png)

**Comment by lee-soft** [comment]
Does the start menu function as normal despite showing the error?

**Comment by GeorgeK1ng** [comment]
Yes, ViStart keeps working when you click on error message. Error is gone and you can use app. But on each WIN key pressed or lick on Orb generate message again.
